code/game/objects/items/weapons/implants/implantchair.dm

```diff
@@ -126,7 +126,7 @@
 
 
 /obj/machinery/implantchair/proc/add_implants()
-	for(var/i=0, i<src.max_implants, i++)
+	for(var/i = 0, i< max_implants, i++)
 		var/obj/item/implant/mindshield/I = new /obj/item/implant/mindshield(src)
 		implant_list += I
```

Comment:
would you make this for loop use the `var/i in 1 to max_implants` syntax

Reply:
done